source/test/UnitTests/Event.cpp

```diff
@@ -0,0 +1,33 @@
+#include "pch.h"
+#include "event.h"
+
+namespace PlayFab
+{
+namespace UnitTests
+{
+
+Event::Event() noexcept
+{
+    m_handle = CreateEvent(nullptr, false, false, nullptr);
+    assert(m_handle);
+}
+
+Event::~Event() noexcept
+{
+    CloseHandle(m_handle);

```

Comment:
for extra safety, might as well do:

if( handle != nullptr ) 
{
    CloseHandle(m_handle);
}